isign/isign.py

```diff
@@ -26,7 +27,7 @@
 
 
 class NotSignable(Exception):
-    """ This is just so we don't expose isign.app.NotSignable """
+    """ This is just so we don't expose other sorts of exceptions """
```

Comment:
You've got a good Quality Day candidate there ;)
On Mar 3, 2016 5:53 PM, "Neil Kandalgaonkar" <notifications@github.com>
wrote:

> In isign/isign.py
> <https://github.com/saucelabs/isign/pull/17#discussion_r54980709>:
>
> > @@ -26,7 +27,7 @@
> >
> >
> >  class NotSignable(Exception):
> > -    """ This is just so we don't expose isign.app.NotSignable """
> > +    """ This is just so we don't expose other sorts of exceptions """
>
> I'm going to keep it as is, b/c there's code in Pantry that expects
> isign.NotSignable, not isign.exceptions.NotSignable.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/saucelabs/isign/pull/17/files#r54980709>.
>